### ISSUE TITLE
Feature: Publish how crypto tax software can access view keys with users' permissions, for logging txns, gains, losses

### ISSUE THREAD

**Issue by crProductGuy** [iron-fish/ironfish#2807]
### Description

Interfacing to Iron Fish's View keys (with user opt-in) by popular multi-nationally-compliant crypto tax software / SaaS packages like Koinly, Coin Tracker, ZenLedger, etc., will be essential for tax compliance by users in many countries. 

I believe that opening discussions with tax vendors early after mainnet release (or even before) and getting on-the record support in place during 2023 will be important to avoid an adoption barrier (uncertainty by users about how to be tax-compliant).

This would apply to both native IF and wrapped assets.

**Comment by NullSoldier** [comment]
Yeah, this is a good point but largely depends on how many custody solutions support Iron Fish. This will largely be supported through your custody solution, like Robinhood, Metamask, or Coinbase who build these features.  I don't know many tax software that analyze a node directly. So if we garner support from those wallets, their tax support will most likely extend to us as well.
